api/v1beta1/groupversion_info.go

```diff
@@ -1,18 +1,38 @@
-/*
-Copyright The Cryostat Authors
-
-Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
@ebaron should these have been Apache licensed to begin with?

Reply:
I was not sure about this so I replaced it for review. I could revert it if it is intentional here :D Also I wonder why we have Apache license here tho?

Reply:
These files are generated by the Operator SDK. The `operator-sdk init` command has a `--license` parameter, which defaults to Apache 2.0:
```
--license string          license to use to boilerplate, may be one of 'apache2', 'none' (default "apache2")
```
We should be fine to change these headers then.